services/datamanager/builtin/sync_test.go

```diff
@@ -22,18 +23,13 @@ import (
 )
 
 const (
-	fiftyMillis = 0.0008
-	syncTime    = time.Millisecond * 250
-)
-
-var (
-	testLastModifiedMillis = 10
+	syncIntervalMillis = 0.0008
```

Comment:
To me this makes it sound like milliseconds is the unit for this variable when really it's in minutes - could we just call it `syncIntervalMins` and the variable below `syncIntervalMillis`?

Reply:
Good call. Updated to syncIntervalMins, but kept below un-united since it's already a time.Duration